utils/frame/remote-externalities/Cargo.toml

```diff
@@ -19,6 +19,7 @@ jsonrpsee-ws-client = { version = "0.3.1", default-features = false, features =
 jsonrpsee-proc-macros = "0.3.1"
 
 env_logger = "0.9"
+frame-support = { path = "../../../frame/support" }
```

Comment:
whats this?

Reply:
while running "cargo +nightly test --all-feature"  I got  "failed to resolve: use of undeclared crate or module `frame_support`
   --> utils/frame/remote-externalities/src/lib.rs:644:3"
so I added "frame-suppport" by path to the Cargo.toml

Reply:
This should only be required when the `remote-test` feature is enabled, so I would put this as an optional dependency and only enable it when the `remote-test` feature is enabled.